frontend/src/pages/MainPage.tsx

```diff
@@ -5,55 +5,40 @@ import Header from '~/components/@common/Header';
 import Map from '~/components/@common/Map';
 import CategoryNavbar from '~/components/CategoryNavbar';
 import CelebDropDown from '~/components/CelebDropDown/CelebDropDown';
-import MapModal from '~/components/MapModal/MapModal';
 import RestaurantCard from '~/components/RestaurantCard';
 import RESTAURANT_CATEGORY from '~/constants/restaurantCategory';
 import { CELEBS_OPTIONS } from '~/constants/celebs';
 import useFetch from '~/hooks/useFetch';
-import useMapModal from '~/hooks/useMapModal';
 import getQueryString from '~/utils/getQueryString';
 import { FONT_SIZE } from '~/styles/common';
+
 import type { Celeb } from '~/@types/celeb.types';
 import type { RestaurantData } from '~/@types/api.types';
-import type { Coordinate, CoordinateBoundary } from '~/@types/map.types';
-import type { Restaurant, RestaurantCategory, RestaurantModalInfo } from '~/@types/restaurant.types';
+import type { CoordinateBoundary } from '~/@types/map.types';
+import type { RestaurantCategory } from '~/@types/restaurant.types';
 
 function MainPage() {
-  const [currentRestaurant, setCurrentRestaurant] = useState<RestaurantModalInfo | null>(null);
-  const { modalOpen, isVisible, closeModal, openModal } = useMapModal(true);
   const [isMapExpanded, setIsMapExpanded] = useState(false);
   const [data, setData] = useState<RestaurantData[]>([]);
   const [boundary, setBoundary] = useState<CoordinateBoundary>();
   const [celebId, setCelebId] = useState<Celeb['id']>(-1);
   const [restaurantCategory, setRestaurantCategory] = useState<RestaurantCategory>('전체');
+  const [isHoveredList, setIsHoveredList] = useState<{ cardId: number; isHovered: boolean }[]>([]);
```

Comment:
제 생각이지만... 상태를 이렇게 두지 않고 그냥 hover 된 cardId 를 넘기는 식으로 하면 로직이 더 간단해 질 것 같은데요?
네이밍이 이건 호버된 리스트니? 의 형식인것도 살짝 어색한 것 같아요..!
고친다면 아래와 같이 고칠 것 같아요!

```suggestion
  const [hoveredCardId, setHoveredCardId] = useState<number | null>(null);
```

그럼 fetch 함수 안에서의 set 로직이 사라지게 될 것 같아요!
```
      const newIsHoveredList = content?.map(restaurant => ({ cardId: restaurant.id, isHovered: false }));
      setIsHoveredList(newIsHoveredList);
```

Reply:
좋습니다! 확실히 더 좋은 로직인 것 같아요 👍